### PR TITLE
Replace build scripts by a single Makefile

### PR DESCRIPTION
**Pull request by hollie**
Hello Rod,

a simple pull request that adds a Makefile instead of the two build scripts.

I've tested the OS X one and it is working as expected.

I have also updated the .gitignore files so that the build outputs are not taken into consideration by the version control system.

I hope this helps you when you are working on this software.

Have a nice day,
 Lieven.

### CODE REVIEW

**Comment by fastrgv** [comment]
This raises some complex issues with me.  I don't mind you adding a
Makefile but I want to keep scripts too.
I might also eventually use gpr build scripts.

My goal is to ensure anyone with linux or osx can run executables I
deliver, (not necessarily what is best for developers).  This means I do
not want to require them to load any compiler or non-standard 3rd party
libraries...GAOL for example.

I am trying to hide the fact that I have an ada compiler installed.  I want
to make sure that all codes run on any target machine whether or not ada is
installed.  I want to deliver a self-contained environment with all runtime
requirements satisfied.  Thus to ensure I include all the runtime libraries
and other requirements I do not have any compiler or libraries in my path.

So, long story short, please keep my scripts.

Rod
